tasks/debian.yml

```diff
@@ -0,0 +1,40 @@
+- name: "Install exim (exim4) the mail transfer agent"
+  apt: >
+    name=exim4-daemon-heavy
```

Comment:
Maybe make it a variable? `exim_flavor=light` in defaults and `name="exim4-daemon-{{exim_flavor}}"` ?